app/controllers/api/v1/search_controller.rb

```diff
@@ -522,29 +525,34 @@ def self.escape_terms_for_regex(term_list:)
       def self.generate_mongo_query_by_context(terms:, base_studies:, accessions:, query_context:)
         case query_context
         when :keyword
-          author_match_study_ids = Author.where(:$text => {:$search => terms}).pluck(:study_id)
+          author_match_study_ids = Author.where(:$text => { :$search => terms }).pluck(:study_id)
```

Comment:
This method is getting more complex -- let's move `generate_mongo_query_by_context` to `app/lib/study_search_service`